packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -229,10 +229,23 @@ const CellCore = (props: CellCoreProps): React.ReactElement => {
     );
 };
 
+const cellTokens = {
+    hover: {
+        background: color.fadedBlue8,
+    },
+    press: {
+        background: color.fadedBlue8,
+    },
+    selected: {
+        background: color.fadedBlue8,
+        foreground: color.activeBlue,
+    },
+};
```

Comment:
The semantic colors from our current action tokens don't apply to the cell:
- the cell hover and press background is different
- there is no selected state for actions currently

I've created this set of tokens for now so that we can swap it out later to support theming. All the usage of primitive colors in this component are consolidated to this object now! We could create new global semantic tokens for this, though I'm not sure if it would be useful for other things (this may be a one-off)

 Let me know if there's something else I can do to help support the token work.